osu.Game.Modes.Taiko/Objects/Drawable/Pieces/CentreHitCirclePiece.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using OpenTK;
+
+namespace osu.Game.Modes.Taiko.Objects.Drawable.Pieces
+{
+    /// <summary>
+    /// A circle piece which is used to visualise CentreHit objects.
+    /// </summary>
+    public class CentreHitCirclePiece : CirclePiece
+    {
+        protected override Framework.Graphics.Drawable CreateIcon()
+        {
+            return new CircularContainer
+            {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Size = new Vector2(45f),

```

Comment:
but it's a circle in a circle that is relative to the size of the outer circle.

go try and resize a taiko `DrawableHit` by changing the top-level size and watch this break (it will not size accordingly)